services/user_service.py

```
@@ -0,0 +1,67 @@
+from common.config import settings
+from common.security import hash_password, encode_id
+from repositories.user_repository import UserRepository
+from schemas.user import SignupRequest
+
+UPLOAD_DIR = settings.upload_dir
+
+if not os.path.exists(UPLOAD_DIR):
+    os.makedirs(UPLOAD_DIR)
+
+class UserService:
+    def __init__(self,user_repo: UserRepository):
```

Ruff (0.14.11)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)